docs/family-pub-key.rst

```diff
@@ -0,0 +1,108 @@
+******************************
+Public Key Transaction Family
+******************************
+
+Overview
+========
+
+The *Public Key* transaction family provides a trustless way to store the public key based on RSA cryptosystem on the REMchain.
```

Comment:
We will also support ECDSA keys as well, 
so probably, let's not concretize RSA algorithm